update.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python
+import glob
+import json
+import os
+import shutil
+import sys
+import time
+try:
+    import urllib.request as request
+except:
+    import urllib as request
+import xml.etree.ElementTree as ET
+import re
+from datetime import datetime
+import locale
+from zipfile import ZipFile
+from argparse import ArgumentParser
+try:
+    input = raw_input
+except NameError:
+    pass
+
+locale.setlocale(locale.LC_ALL, '')
+
+NIGHTLY_PATH = 'https://dl.bintray.com/cometvisu/CometVisu/'
+RELEASE_PATH = 'https://api.github.com/repos/CometVisu/CometVisu/releases/latest'
+date_format = '%Y-%m-%dT%H:%M:%SZ'
+
+sep_width = 70
+
+
+def get_latest_nightly():
+    response = request.urlopen(NIGHTLY_PATH)
```

Comment:
Running the code manually line by line here the exception is happening:
```
>>> response = request.urlopen(NIGHTLY_PATH)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/urllib.py", line 86, in urlopen
    return opener.open(url)
  File "/usr/lib/python2.7/urllib.py", line 207, in open
    return getattr(self, name)(url)
  File "/usr/lib/python2.7/urllib.py", line 436, in open_https
    h.endheaders(data)
  File "/usr/lib/python2.7/httplib.py", line 954, in endheaders
    self._send_output(message_body)
  File "/usr/lib/python2.7/httplib.py", line 814, in _send_output
    self.send(msg)
  File "/usr/lib/python2.7/httplib.py", line 776, in send
    self.connect()
  File "/usr/lib/python2.7/httplib.py", line 1161, in connect
    self.sock = ssl.wrap_socket(sock, self.key_file, self.cert_file)
  File "/usr/lib/python2.7/ssl.py", line 381, in wrap_socket
    ciphers=ciphers)
  File "/usr/lib/python2.7/ssl.py", line 143, in __init__
    self.do_handshake()
  File "/usr/lib/python2.7/ssl.py", line 305, in do_handshake
    self._sslobj.do_handshake()
IOError: [Errno socket error] [Errno 104] Connection reset by peer
```
As this is a server response the client can't do much about it. 

Running the same lines on my kubuntu box it's working without exception...

Trying to debug it further on the WireGate doesn't give further insight:
```
http_logger = urllib2.HTTPSHandler(debuglevel = 1)
opener = urllib2.build_opener(http_logger)
urllib2.install_opener(opener)
request = urllib2.Request('https://dl.bintray.com/cometvisu/CometVisu/')
response = urllib2.urlopen(request)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/urllib2.py", line 126, in urlopen
    return _opener.open(url, data, timeout)
  File "/usr/lib/python2.7/urllib2.py", line 400, in open
    response = self._open(req, data)
  File "/usr/lib/python2.7/urllib2.py", line 418, in _open
    '_open', req)
  File "/usr/lib/python2.7/urllib2.py", line 378, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 1215, in https_open
    return self.do_open(httplib.HTTPSConnection, req)
  File "/usr/lib/python2.7/urllib2.py", line 1177, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [Errno 104] Connection reset by peer>
```